common/app/views/support/Profile.scala

```diff
@@ -162,23 +162,23 @@ object ImgSrc extends Logging with implicits.Strings {
       .map(browserWidth => Profile(width = Some(browserWidth), hidpi = hidpi, isPng = isPng))
       .map { profile => {
         maybePath
-          .map(url => srcsetForProfile(profile, url))
-          .orElse(maybeImageMedia.map(imageContainer => srcsetForProfile(profile, imageContainer)))
+          .map(url => srcsetForProfile(profile, url, hidpi))
+          .orElse(maybeImageMedia.map(imageContainer => srcsetForProfile(profile, imageContainer, hidpi)))
           .getOrElse("")
       } }
       .mkString(", ")
   }
 
-  def srcsetForProfile(profile: Profile, imageContainer: ImageMedia): String = {
+  def srcsetForProfile(profile: Profile, imageContainer: ImageMedia, hidpi: Boolean): String = {
     if(ImageServerSwitch.isSwitchedOn) {
-      s"${findLargestSrc(imageContainer, profile).get} ${profile.width.get}w"
+      s"${findLargestSrc(imageContainer, profile).get} ${profile.width.get * (if (hidpi) 2 else 1)}w"
     } else {
-      s"${findNearestSrc(imageContainer, profile).get} ${profile.width.get}w"
+      s"${findNearestSrc(imageContainer, profile).get} ${profile.width.get * (if (hidpi) 2 else 1)}w"
     }
   }
 
-  def srcsetForProfile(profile: Profile, path: String): String = {
-    s"${ImgSrc(path, profile)} ${profile.width.get}w"
+  def srcsetForProfile(profile: Profile, path: String, hidpi: Boolean): String = {
+    s"${ImgSrc(path, profile)} ${profile.width.get * (if (hidpi) 2 else 1)}w"
```

Comment:
if you wanted to avoid repeating the 2 and 1 you could create a trait DPI with a field `multiplier` and inherit to case objects LoDpi and HiDpi which have 2 and 1.  However the level of repetition isn't amazingly high so you could go either way on that.